dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java

```diff
@@ -204,7 +204,7 @@ public void testNotKnownEndpoint() throws Exception {
 
     @Test
     @SkipForRepeat({ TelemetryActions.MP14_MPTEL20_ID, TelemetryActions.MP41_MPTEL20_ID, TelemetryActions.MP50_MPTEL20_ID, TelemetryActions.MP50_MPTEL20_JAVA8_ID,
```

Comment:
Can SkipForRepeat take on an array? If it does, it is better to define a array to contain the individual enums